### PR TITLE
Implemented Unit Testing, Cypress E2E Environment 

### PR DESCRIPTION
**Pull request by apark105**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Here is a set of instructions that will get the project up and running on your local machine for development and testing purposes.
+
+1. Fork this repo 
+
+2. Git clone 
+
+3. 
+APP
+```
+cd App
```
Comment (l1fb) [inline README.md]: do these commands match exactly what people can use? If they were to copy paste these commands, would they run?

Comment (l1fb) [reply README.md]: if it does, could you lower case the `Test` folder?

Comment (l1fb) [reply README.md]: and the `App` folder

Comment (l1fb) [reply README.md]: actually app folder is fine

Comment (l1fb) [reply README.md]: Tests -> test

Comment (apark105) [reply README.md]: To answer the first question, yes, they can. Also, I am going to keep App and Test both uppercase just for consistency.

**Pull request merged**